boards/beagle/beaglebone_ai64/beaglebone_ai64_j721e_r5.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2023 BeagleBoard.org Foundation
+# Copyright (C) 2023 S Prashanth
+#
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: beaglebone_ai64/j721e/r5
+name: BeagleBone-AI64 R5
+type: mcu
+arch: arm
+ram: 32
```

Comment:
ATCM and BTCM should be 64.

Reply:
The BTCM is at a non-contiguous address from the ATCM. I've not found a good way without really messing with the linker files to use both. Do you know of some standard way to have the binary sections split across two memories like that?

If not then we only really have 32KB RAM usable (until I add the DDR section).

Reply:
Do we need to add virtual memory on top of both sections? I'm not familiar with that too. Might be easier to use DDR and just ignore BTCM.

The documentation mentions 64kB as available memory. Maybe this should be 32, too?

Reply:
The documentation is combining both 32KB ATCM and 32KB BTCM.

I'm planning on going with DDR as the "main" memory for this core (which would mean 16MB as that is how much we carveout for each R5 core today from Linux). But to use that we will want to turn on caching (DDR is far away from the core) and add RAT support which is going to be added in the next PR. Wanted to keep this PR simple.